starboard/loader_app/BUILD.gn

```diff
@@ -267,30 +273,29 @@ if (!is_host_win) {
   }
 }
 
-static_library("slot_management") {
-  sources = [
-    "slot_management.cc",
-    "slot_management.h",
-  ]
-  deps = [
-    ":app_key_files",
-    ":drain_file",
-    ":installation_manager",
-    "//starboard:starboard_group",
-    "//starboard/elf_loader",
-    "//starboard/elf_loader:constants",
-    "//starboard/elf_loader:sabi_string",
-  ]
-
-  if (sb_is_evergreen_compatible && current_toolchain == starboard_toolchain) {
-    deps += [ "//third_party/crashpad/wrapper" ]
-  } else {
-    deps += [ "//third_party/crashpad/wrapper:wrapper_stub" ]
+if (!sb_is_modular && sb_is_evergreen_compatible &&
+    current_toolchain == starboard_toolchain) {
+  static_library("slot_management") {
+    sources = [
+      "slot_management.cc",
+      "slot_management.h",
+    ]
+    deps = [
+      ":app_key_files",
+      ":drain_file",
+      ":installation_manager",
+      "//starboard:starboard_group",
+      "//starboard/elf_loader",
+      "//starboard/elf_loader:constants",
+      "//starboard/elf_loader:sabi_string",
+      "//third_party/crashpad/wrapper",
+    ]
   }
 }
 
 # TODO: b/309493306 - Stop building evergreen targets for all non-evergreen platforms.
```

Comment:
should we remove TODO's in all cases ? it's no longer needed

Reply:
Removed